databricks_cli/dbfs/api.py

```diff
@@ -45,29 +45,39 @@ class ParseException(Exception):
 
 
 class FileInfo(object):
-    def __init__(self, dbfs_path, is_dir, file_size):
+    def __init__(self, dbfs_path, is_dir, file_size, modification_time):
         self.dbfs_path = dbfs_path
         self.is_dir = is_dir
         self.file_size = file_size
+        self.modification_time = modification_time
 
     def to_row(self, is_long_form, is_absolute):
         path = self.dbfs_path.absolute_path if is_absolute else self.dbfs_path.basename
         stylized_path = click.style(path, 'cyan') if self.is_dir else path
         if is_long_form:
             filetype = 'dir' if self.is_dir else 'file'
-            return [filetype, self.file_size, stylized_path]
+            # Do not return modification time if it is not available from server.
+            if self.modification_time is None:
+                return [filetype, self.file_size, stylized_path]
+            else:
+                return [filetype, self.file_size, stylized_path, self.modification_time]
```

Comment:
nit: Another way in which you could do this is the following:
```
row = [filetype, self.file_size, stylized_path]
if self.modification_time is not None:
  row.append(self.modification_time)
return row
```
This avoids duplicating the code and scales if we want to add more optional parameters. But I don't have a strong opinion about it, it's just a matter of style.